model/tps/cics-spans.yaml

```diff
@@ -0,0 +1,28 @@
+groups:
+  - id: span.ibm.cics.http.server
+    type: span
+    stability: development
+    brief: 'This group defines semantic conventions for a HTTP server spans of IBM CICS Transaction Server for z/OS.'
+    extends: span.tps.http.server
+    span_kind: server
+    attributes:
+      - ref: tps.system.name
+        requirement_level: required
+        examples: 'ibm.cics'
+      - ref: ibm.cics.transaction.group.id
+
+  - id: span.ibm.cics.rpc.server
+    type: span
+    stability: development
+    brief: 'This group defines semantic conventions for Remote Procedure Call (RPC) server spans of IBM CICS Transaction Server for z/OS.'
+    extends: span.tps.rpc.server
+    span_kind: server
+    attributes:
+      - ref: tps.system.name
+        requirement_level: required
+        examples: 'ibm.cics'
+      - ref: network.protocol.name
+        examples: ['ipic', 'mro']
+      - ref: rpc.system
+        examples: ['dpl', 'rmi']
```

Comment:
FYI: this attribute will be renamed and deprecated in scope of https://github.com/open-telemetry/semantic-conventions/issues/2703. Also neither dpl nor rmi are defined yet - we'd need to eventually define and document both.